nengo/utils/numpy.py

```diff
@@ -251,7 +251,7 @@ def rmse(x, y, axis=None, keepdims=False):  # pragma: no cover
 
 if hasattr(np.fft, "rfftfreq"):
     rfftfreq = np.fft.rfftfreq
-else:
+else:  # pragma: no cover
```

Comment:
We might want to monkeypatch into here just so that we can test that our `rfftfreq` does the same as Numpy's.